app/controllers/PlutoController.scala

```diff
@@ -1,17 +1,29 @@
 package controllers
 
+import java.util.Locale
+import com.gu.media.aws._
+import com.gu.media.Settings
+import com.gu.media.logging.Logging
 import com.gu.media.pluto.PlutoUpsertRequest
+import com.gu.media.upload.PlutoUploadActions
 import com.gu.pandahmac.HMACAuthActions
+import com.gu.media.util.MediaAtomHelpers
 import data.{DataStores, UnpackedDataStores}
 import play.api.libs.json.Json
 import play.api.mvc.Controller
+import com.gu.media.model.{MediaAtom, MediaAtomBeforeCreation, PlutoResyncMetadataMessage}
+import com.typesafe.config.{Config, ConfigFactory}
+import util.AWSConfig
 
 class PlutoController(
+  val config:Config,
+  val awsConfig:AWSConfig,
   val authActions: HMACAuthActions,
   override val stores: DataStores
 ) extends Controller
   with UnpackedDataStores
-  with JsonRequestParsing {
+  with JsonRequestParsing
```

Comment:
do we need to extend with `JsonRequestParsing`?

Reply:
in a word.... yes.

```
[error] /Users/localhome/workdev/media-atom-maker/app/controllers/PlutoController.scala:39: object parse does not take type parameters.
[error]     parse[PlutoUpsertRequest](req) { data: PlutoUpsertRequest => {
[error]          ^
[error] one error found
[error] (app/compile:compileIncremental) Compilation failed
[info] Compiling 1 Scala source to /Users/localhome/workdev/media-atom-maker/target/scala-2.11/classes...
[error] /Users/localhome/workdev/media-atom-maker/app/controllers/PlutoController.scala:39: object parse does not take type parameters.
[error]     parse[PlutoUpsertRequest](req) { data: PlutoUpsertRequest => {
[error]          ^
[error] one error found
[error] (app/compile:compileIncremental) Compilation failed
```